target/linux/mediatek/image/mt7622.mk

```diff
@@ -270,6 +270,34 @@ define Device/ubnt_unifi-6-lr-v1-ubootmod
 endef
 TARGET_DEVICES += ubnt_unifi-6-lr-v1-ubootmod
 
+define Device/ubnt_unifi-6-lr-v2
+  DEVICE_VENDOR := Ubiquiti
+  DEVICE_MODEL := UniFi 6 LR v2
+  DEVICE_DTS_CONFIG := config@1
+  DEVICE_DTS := mt7622-ubnt-unifi-6-lr-v2
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_PACKAGES := kmod-mt7915e
+endef
+TARGET_DEVICES += ubnt_unifi-6-lr-v2
+
+define Device/ubnt_unifi-6-lr-v2-ubootmod
+  DEVICE_VENDOR := Ubiquiti
+  DEVICE_MODEL := UniFi 6 LR v2
```

Comment:
Also here, `v2` should rather be put in `DEVICE_VARIANT`.

Reply:
thanks fixing